### DISCUSSION TITLE
How is new image loaded so fast ? I want the same thing while using tabliss.

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [Floorp-Projects/Floorp#475] [Q&A] [answered]
What is the implementation behind image loading ? is it prefetched ? or fetched on the fly ?

**Comment by Ryosuke-Asano** [comment, marked as answer]
Maybe you post this question on Discord. I answered.

**Reply by KAGEYAM4** [reply]
yes , sorry i didn't get any notification  so i thought the post got missed.

Thank you. 

sorry you had to answer twice.

**Reply by Ryosuke-Asano** [reply]
Ok
